en/docs/install-and-setup/setup/api-controller/managing-common-api-policies/managing-common-api-policies.md

```diff
@@ -0,0 +1,114 @@
+                **Flags:**  
+                
+                -   Required :  
+                    `--environment` or `-e` : Environment to be searched  
+                -   Optional :
+                    `--all`: All Common API Policies available in the environment
+                    `--limit` or `-l` : Maximum number of Common API Policies to return (Default 25)
+                    `--format` : pretty-print environments using templates
+
+            !!! example
+                ```bash
+                apictl get policies api -e dev 
+                ```
+                ```bash
+                apictl get policies api --environment production --limit 15 
+                ```    
+                ```bash
+                apictl get policies api --environment production
+                ```  
+                ```go
+                apictl get policies api --environment production --all 
+                ```  
+
+        -   **Response**
+
+            ```go
+            ID                                     NAME                       Display NAME               CATEGORY            APPLICABLE FLOWS     SUPPORTED GATEWAYS
+            e758ccd3-39ec-430c-8722-a44253a53ecd   ccCallInterceptorService   Call Interceptor Service   Mediation           [request response]   [ChoreoConnect]
+            55e8e47f-35ee-46b3-b097-5f347079c7c4   setToHeader                Set To Header              Mediation           [request]            [Synapse]
+            ```
+            
+            !!! tip 
+                When using the `apictl get policies api -e dev` command, `--all` optional flag can be used to 
+                get all available Common API Policies.
+
+            !!!note
+                Output of the `get policies api` command can be formatted with Go Templates. For more information on formatting the get commands, see [Formatting the outputs of get commands]({{base_path}}/install-and-setup/setup/api-controller/advanced-topics/formatting-the-output-of-get-command).
+               
+
+## Delete a Common API Policy in an environment
+
+Follow the instructions below to delete a Common API Policy in an environment using apictl:
+
+1.  Make sure that the WSO2 API-M 4.1.0 version is started and that the 4.1.0 version of apictl is set up.   
+For more information, see [Download and Initialize the apictl]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-apictl).
+2.  Log in to the WSO2 API-M in the environment by following the instructions in [Login to an Environment]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#login-to-an-environment).
+3.  Run the corresponding apictl command below to delete a Common API Policy in an environment.
+
+    1. Delete a Common API Policy in an environment.
+
+        -   **Command**
+            ``` bash
+            apictl delete policy api -n <Common API Policy name> -e <environment>
```

Comment:
Let's include the type flag in the doc too.

Reply:
Use the format <Common API Policy-name>
Here, note that "Common API Policy" is the artifact name, so we can keep that as it is. "name" is an attribute of the common API policy, so we need to use a hyphen to append it to the main artifact.
Let's follow this format everywhere. Refer [1] for more information.

[1] https://apim.docs.wso2.com/en/4.1.0/install-and-setup/setup/api-controller/managing-apis-api-products/migrating-api-products-to-different-environments/

Reply:
You mean we have to add examples for `--name` as well right?

Reply:
No changing the flag <Common API Policy name> to <Common API Policy-name>

Reply:
Fixed by [14ca1f8](https://github.com/wso2/docs-apim/pull/6103/commits/14ca1f88a1c77bb52d4234c692aef6534f17f45c)